packages/extension-chakra-storefront/src/components/with-multi-site/with-multi-site.tsx

```diff
@@ -16,19 +16,27 @@ import {MultiSiteProvider} from '../../contexts'
 import {resolveSiteFromUrl, resolveLocaleFromUrl} from '../../utils/site-utils'
 import {useExtensionConfig} from '../../hooks/use-extension-config'
 
+interface Request {
+    originalUrl?: string
+}
```

Comment:
Is this request type a node request type.. should we be redefining the type here?

Reply:
I added a new type to commerce-sdk-react